packages/@coorpacademy-components/src/organism/setup-header/style.css

```diff
@@ -208,33 +187,18 @@
     border-bottom: 8px solid cm_grey_100;
   }
 
-  [data-name="logout-show"] .logOut {
-    display: flex;
-    margin: 0;
-    color: black;
-  }
-
-  [data-name="logout-show"] .logOut:hover {
```

Comment:
je ne savais pas qu'on pouvait utiliser le data-name comme selector haha the more you know